schema/course.model.schema

```diff
@@ -361,7 +361,7 @@
               "type": "string",
               "title": "Alternative Feedback Title",
               "help": "Text read out by screen readers if no visual title is set for component question feedback.",
-              "default": "Feedback",
+              "default": "",
```

Comment:
I think this should probably stay to make sure there is always something in the absence of a given feedback title, component title or specified alt title. The notify dialog must have some specified accessible name.

Reply:
Ok, that makes sense. I'll rework this.